include/ivi-logging-file.h

```diff
@@ -35,9 +35,17 @@ class FileLogContext : public StreamLogContextAbstract {
     FileLogContext(){toFile=true;}
 
 private:
+    struct FileWrapper{
+        FILE* m_file=nullptr;
```

Comment:
to be on the safe side you should delete copy constr. and assignment operator.

Reply:
Fixed on #7